src/containers/sidebar/sidebar.component.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Navbar from "containers/navbar";
+import { Route, Link, BrowserRouter as Router } from "react-router-dom";
+import { useSelector } from "react-redux";
+import { auth } from "database";
+const Sidebar = (props) => {
+  const currentUser = useSelector((state) => state.users.user);
+  const signOut = () => {
+    auth.signOut();
+    props.logout();
+  };
+  return (
+    <div>
+      <Navbar navigation="/" postAdd="home" />
```

Comment:
I would try and think of another for `postAdd` . Best to keep the props of your components very understandable since they are its contract with the outside world

Reply:
Yup, this was on my list of things to change